docs/en/ingest-management/fleet/fleet-settings.asciidoc

```diff
@@ -0,0 +1,94 @@
+[[fleet-settings]]
+= {fleet} settings
+
+beta[]
+
+Configure {fleet} settings in {kib} to apply global settings to all {agent}s
+enrolled in {fleet}:
+
+. In {kib}, open the main menu, then click *Management > {fleet}*.
+
+. Click *{fleet} settings*
++
+--
+[cols="2*<a"]
+|===
+|
+[id="fleet-server-hosts-setting"]
+*{fleet-server} hosts*
+
+| The URLs your {agent}s will use to connect to a {fleet-server}. This setting
+is required. On self-managed clusters, you must specify one or more URLs. On
+{ecloud}, this field is populated automatically.
+
+//QUESTION: What is valid here? URL implies something like http://localhost:8220.
```

Comment:
https is the only supported protocol... I think.

I am uncertain what our intentions are with regards to error handling and defaults... if the port is left off it was broken at one point, so perhaps the intention is that it is not explicitly required.  I'd want to see @nchaulet confirmation here on what we ended up with and then we could document leaving off port and/or alternate port usage.

Reply:
If the port is not set we will automatically populate in Kibana to the default for http or https

Reply:
@nchaulet So it sounds like `http` and `https` are both supported. Is 80 just an example or the actual default?

Reply:
it's the default for http and `443` is the default for https